cdhweb/static_src/global/mixins/full-bleed-bg.scss

```diff
@@ -0,0 +1,6 @@
+// Full-bleed background :)
+// https://www.smashingmagazine.com/2024/01/css-border-image-property/#full-width-backgrounds
+// Note the 2nd 0 in `0/0/0` is only needed because otherwise Sass sees `//` and comments out the rest of the line :/
+@mixin full-bleed-bg($color) {
+  border-image: conic-gradient($color 0 0) fill 0/0/0 100vw;
```

Comment:
excited to see someone have a crack at using this wacky syntax! (I'm also glad it wasn't me)

Reply:
honestly this is a game changer!!

Reply:
Oh true, good call! Didn't know we could do it like that. I'll definitely make this change at some stage.